wolfcrypt/src/asn.c

```diff
@@ -14806,11 +14811,15 @@ int ExtractDate(const unsigned char* date, unsigned char format,
     int tm_min  = certTime->tm_min;
     int tm_sec  = certTime->tm_sec;
 
-#ifdef WOLFSSL_LINUXKM
-    if (GetTime_Long(&tm_year, date, idx) != 0) return 0;
-#else
-    if (GetTime(&tm_year, date, idx) != 0) return 0;
-#endif
+    #ifdef WOLFSSL_LINUXKM
```

Comment:
If this section is "AVR" then `WOLFSSL_LINUXKM` will never be used here, please remove it.

Don't just add braces around the expression and return. Instead format each one like this:

```
if (GetTime(&certTime->tm_year, date, idx) != 0) {
    return 0;
}
````

Reply:
I've removed the `WOLFSSL_LINUXKM` for AVR that was previously there, but I left the if-without-brackets as that was already there. I can cleanup all the instances if desired.